src/current/molt/migrate-failback.md

```diff
@@ -223,40 +229,42 @@ Create a CockroachDB changefeed to send changes to MOLT Replicator.
       1759246920563173000.0000000000
     ~~~
 
-1. Create the CockroachDB changefeed pointing to the MOLT Replicator webhook endpoint. Use `cursor` to specify the logical timestamp from the preceding step.
+1. Create the CockroachDB changefeed pointing to the MOLT Replicator webhook endpoint. Use `cursor` to specify the logical timestamp from the preceding step. For details on the webhook sink URI, refer to [Webhook sink]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-sink).
 
     {{site.data.alerts.callout_info}}
-    Ensure that only **one** changefeed points to MOLT Replicator at a time to avoid mixing streams of incoming data.
-    {{site.data.alerts.end}}
-
-    {{site.data.alerts.callout_success}}
-    For details on the webhook sink URI, refer to [Webhook sink]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-sink).
+    Explicitly set a default `10s` [`webhook_client_timeout`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#options) value in the `CREATE CHANGEFEED` statement. This value ensures the webhook can report failures in inconsistent networking situations, and make crash loops more visible.
     {{site.data.alerts.end}}
 
     <section class="filter-content" markdown="1" data-scope="postgres">
+    The target schema is specified in the webhook URL path in the fully-qualified format `/database/schema`. The path specifies the database and schema on the target PostgreSQL database. For example, `/molt/migration_schema` routes changes to the `migration_schema` schema in the `molt` database.
+
     {% include_cached copy-clipboard.html %}
     ~~~ sql
     CREATE CHANGEFEED FOR TABLE employees, payments, orders \
-    INTO 'webhook-https://replicator-host:30004/migration_schema/public?client_cert={base64_encoded_cert}&client_key={base64_encoded_key}&ca_cert={base64_encoded_ca}' \
-    WITH updated, resolved = '250ms', min_checkpoint_frequency = '250ms', initial_scan = 'no', cursor = '1759246920563173000.0000000000', webhook_sink_config = '{"Flush":{"Bytes":1048576,"Frequency":"1s"}}';
+    INTO 'webhook-https://replicator-host:30004/molt/migration_schema?client_cert={base64_encoded_cert}&client_key={base64_encoded_key}&ca_cert={base64_encoded_ca}' \
+    WITH updated, resolved = '250ms', min_checkpoint_frequency = '250ms', initial_scan = 'no', cursor = '1759246920563173000.0000000000', webhook_sink_config = '{"Flush":{"Bytes":1048576,"Frequency":"1s"}}', webhook_client_timeout = '10s';
     ~~~
     </section>
 
     <section class="filter-content" markdown="1" data-scope="mysql">
+    MySQL tables belong directly to the database, not to a separate schema. The webhook URL path specifies the database name on the target MySQL database. For example, `/public` routes changes to the `public` database.
```

Comment:
Similar comment to above. May be worthwhile to use `molt` for this database name, since public is usually a schema name.